packages/storybook/stories/inbox/full-page-inbox.stories.tsx

```diff
@@ -65,14 +79,14 @@ export const Hooks = () => {
           <ReactMarkdown>{`## Example`}</ReactMarkdown>
           <ReactMarkdown>{`\`\`\`javascript\n${fullPageInboxHooksString}\n\`\`\``}</ReactMarkdown>
         </div>
-        <CourierProvider {...stagingCourierProps}>
+        <CourierProvider {...devCourierProops}>
           <FullPageInboxHooks />
         </CourierProvider>
-        <Frame>
+        {/*<Frame>
```

Comment:
just making sure commenting this out is intended

Reply:
it doesn't really matter as its just the dev enviroment and nobody looks at the storybook